utils/utils.go

```diff
@@ -49,3 +51,11 @@ const (
 func (t TagDataType) String() string {
 	return string(t)
 }
+
+func MakeKeyForRFCache(bucketTs int64, fingerprint string) string {
```

Comment:
repeated in logsexporter , consider removing it from there ?

Reply:
Will do it follow up PR, there is another PR in the line